.vsts-dotnet-ci.yml

```diff
@@ -26,11 +29,19 @@ phases:
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: Linux
+    name: Linux_Centos
     buildScript: ./build.sh
     queue:
       name: Hosted Ubuntu 1604
-      container: LinuxContainer
+      container: LinuxCentosContainer
+
+- template: /build/ci/phase-template.yml
+  parameters:
+    name: Linux_Ubuntu
```

Comment:
Was one of the proposals that we would use `netcoreapp3.0` on Ubuntu?  That way would could get some non-Windows netcoreapp3.0 coverage?